Unit2_Applications/friendship-quiz.js

```diff
@@ -0,0 +1,40 @@
+// questions to be randomly asked on page load
+var questionsList = ["What are your/your friend's favorite animals?", "What do I/you hate the most(foodwise)?", "What is my favorite color?"];
+
+
+//randomly chooses a question to display
+function init() {
+  var randomIndex = Math.floor(Math.random() * questionsList.length);
+  document.getElementById("question").innerHTML = questionsList[randomIndex];
+}
+
+window.onload = init;
+
+var answerList = []; //initialize answerList to be empty
+
+
+//adds user-submitted answer to answerList
+function submitAnswer() {
+  var answer = document.getElementById("answerInput").value;
+  var answer2 = answer.toUpperCase();
+  document.getElementById("answerInput").value = "";
+  answerList.push(answer2);
+  console.log(answerList);
```

Comment:
`console.log(answerList)` is a **great** line to have while you're writing code, but it's not necessary for your final product, so this would be a line to delete when cleaning up your code 😃